python/bicoherence/bicoherence_calculations.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
```

Comment:
Just why is this here, and not in the python-bicoherence branch?

Reply:
I think the initial idea was to have a python branch, with the calculating routines in it. Thus bicoherence ended up here.
I agree, that it should be moved to a separated branch or project (as it was before).